src/main/java/racingcar/domain/CarDistance.java

```diff
@@ -2,17 +2,21 @@
 
 public record CarDistance(int value) {
     private static final int DEFAULT_DISTANCE = 0;
-    private static final int MOVE_DISTANCE = 1;
+    private static final int STEP = 1;
 
     public CarDistance() {
         this(DEFAULT_DISTANCE);
     }
 
-    public int getValue() {
-        return this.value;
+    public CarDistance increase() {
+        return new CarDistance(this.value + STEP);
     }
 
-    public CarDistance increase() {
-        return new CarDistance(this.value + MOVE_DISTANCE);
+    public int max(int that) {
+        return Math.max(this.value, that);
+    }
```

Comment:
```suggestion
    public CarDistance max(int that) {

    }
    
    public CarDistance max(CarDistance that) {

    }
```
- 메서드 반환 값을 int가 아닌 CarDistance로 구현해 고려해 볼 요소 중 하나.
- max 메서드에 int 외에 CarDistance와 같은 타입을 받도록 오버로딩하는 것도 고려해볼 요소 중 하나.

이와 같이 여러 메서드를 오버로딩하면 이 메서드를 호출하는 개발자에게 편리성을 제공해 줄 수 있지 않을까?
한번씩 고려해 보면 좋겠음.